### PR TITLE
Add Package Maintenance WG

### PR DESCRIPTION
**Pull request by rxmarbles**
As of this PR https://github.com/nodejs/TSC/pull/880 the package maintenance WG has been chartered. We would like to add the WG to the NodeJS website

### CODE REVIEW

File: locale/en/about/working-groups.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Core Working Groups are created by the
 * [Release](#release)
 * [Security](#security)
 * [Streams](#streams)
+* [Package Maintenance](#package-maintenance)
```
Comment (Trott) [inline locale/en/about/working-groups.md]: The working groups are listed in alphabetical order so this should be moved up a few lines.

```suggestion
* [Package Maintenance](#package-maintenance)
* [Release](#release)
* [Security](#security)
* [Streams](#streams)
```

File: locale/en/about/working-groups.md [hunk 2]
```diff
@@ -239,3 +240,27 @@ Responsibilities include:
...
+* Building and documenting guidance, tools and processes so businesses
+  can identify the packages they depend on. Businesses can use the information
+  to build a business case which supports both the organization and developers
+  helping to maintain those packages.
+* Documenting a backlog and providing resources to help businesses identify
+  how their developers can contribute, and get engaged. Developers can test
+  and validate a workflow to help with issues slowing migration to Node.js 10.x.
+* Building, documenting and evangelizing guidance, tools and processes
+  (for example LTS for modules) can make it easier for maintainers to manage multiple streams,
+  and accept help from those who depend on their module.
```
Comment (Trott) [inline locale/en/about/working-groups.md]: Same thing about alphabetical order...this section should be moved up a bit.

**Comment by Trott** [comment]
It's possible that listing goals is better than listing responsibilities--I haven't thought about that--but all the other entries list the responsibilities, and those are listed in https://github.com/nodejs/TSC/pull/880. So maybe that's what should be here rather goals?

**Comment by Trott** [comment]
Thanks for doing this. I have comments above, but nothing blocking it landing if others review it.

**Comment by rxmarbles** [comment]
> It's possible that listing goals is better than listing responsibilities--I haven't thought about that--but all the other entries list the responsibilities, and those are listed in [nodejs/TSC#880](https://github.com/nodejs/TSC/pull/880). So maybe that's what should be here rather goals?

Good catch on the responsibilities I missed that PR. Seems it needs a fix though as the package maintenance wg isn't listed in the list of WG's I'll get that updated as well in a fast follow

File: locale/en/about/working-groups.md [hunk 3]
```diff
@@ -177,6 +178,29 @@ Each language community maintains its own membership.
...
+### [Package Maintenance](https://github.com/nodejs/package-maintenance)
+
+Responsibilities include:
+* Building, documenting and evangelizing guidance, tools and processes that make it easier for maintainers to
+  maintain packages and accept help from those who depend on their packages.
+* Management of repositories within the [pkgjs](https://github.com/pkgjs)
+  GitHub organization including but not limited to:
+  * Managing the list of organization owners which supplement the standard
+    Node.js organization owners as outlined in:
+[https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners](https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners)
```
Comment (nschonni) [inline locale/en/about/working-groups.md]: Simpler if you're going for the plan URL
```suggestion
    <https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners>
```

File: locale/en/about/working-groups.md [hunk 4]
```diff
@@ -177,6 +178,29 @@ Each language community maintains its own membership.
...
+  * Overseeing new repositories (creating, moving, removing)
+  * Managing the maintainer teams for all of the repositories.
+  * Contribution policy for repositories
+* Technical direction for the projects within
+  the [pkgjs](https://github.com/pkgjs)  organization
```
Comment (nschonni) [inline locale/en/about/working-groups.md]:
```suggestion
  the [pkgjs](https://github.com/pkgjs) organization
```

File: locale/en/about/working-groups.md [hunk 5]
```diff
@@ -177,6 +178,29 @@ Each language community maintains its own membership.
...
+* Project governance and process (including this policy)
+* Managing the maintainer teams and contribution policies for the
+  following repositories
+  * nodejs/ci-config-travis
+  * nodejs/ci-config-github-actions
+  * nodejs/package-maintenance repository.
+
```
Comment (nschonni) [inline locale/en/about/working-groups.md:201]:
```suggestion
```

File: locale/en/about/working-groups.md [hunk 6]
```diff
@@ -177,6 +178,29 @@ Each language community maintains its own membership.
...
+  GitHub organization including but not limited to:
+  * Managing the list of organization owners which supplement the standard
+    Node.js organization owners as outlined in:
+[https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners](https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners)
+  * Overseeing new repositories (creating, moving, removing)
+  * Managing the maintainer teams for all of the repositories.
+  * Contribution policy for repositories
+* Technical direction for the projects within
+  the [pkgjs](https://github.com/pkgjs)  organization
+* Project governance and process (including this policy)
```
Comment (nschonni) [inline locale/en/about/working-groups.md]: Not really clear what "(including this policy)" is referring to. I'm guessing maybe this text came out of a different document

Comment (mhdawson) [reply locale/en/about/working-groups.md]: I agree that just removing that line makes sense to me

**Review by mhdawson — APPROVED** [review]
LGTM

File: locale/en/about/working-groups.md (hunk 6 above)
Comment (rxmarbles) [reply locale/en/about/working-groups.md]: Sounds good I'll work on getting a PR up to fix the TSC doc this came from @mhdawson

**Review by aymen94 — APPROVED** [review]

**Review by nschonni — APPROVED** [review]

**Comment by rxmarbles** [comment]
@nschonni @aymen94 @mhdawson can we merge this PR? seems it has a decent amount of approvals. Plus I don't want to keep merging from master 😜

**Review by Trott — APPROVED** [review]

**Comment by Trott** [comment]
Might be nice to add a sentence explaining what the working group's function is, like the other ones have, but that can happen in a subsequent PR, or not at all. LGTM. Landing.

**Pull request merged**